gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2698, in get_exp_list
    im = m.get_cutout(i, j, type='image')
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 226, in get_cutout
    imflat = self._fits[extname][start_row:row_end]
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/fitsio-1.0.0rc2-py3.6-linux-x86_64.egg/fitsio/fitslib.py", line 3425, in __getitem__
    return self._read_image_slice(arg)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/fitsio-1.0.0rc2-py3.6-linux-x86_64.egg/fitsio/fitslib.py", line 3436, in _read_image_slice
    return self._read_image_slice((arg,))
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/fitsio-1.0.0rc2-py3.6-linux-x86_64.egg/fitsio/fitslib.py", line 3516, in _read_image_slice
    self._FITS.read_image_slice(self._ext+1, first, last, steps, array)
OSError: FITSIO status = 108: error reading from FITS file
Error reading data buffer from file:
./fiducial_H158_2284539.fits
Error reading elements 1 thru 50176 from image (ffgcld).

Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
gzip: ./fiducial_H158_2284539.fits already exists;	not overwritten
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
Traceback (most recent call last):
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3700, in <module>
    m.get_coadd_shape()
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 3095, in get_coadd_shape
    obs_list,psf_list,included,w = self.get_exp_list(m,ii)
  File "../../wfirst_imsim/wfirst_imsim/simulate.py3", line 2699, in get_exp_list
    im_psf = m2.get_psf(i, j)
  File "/users/PCON0003/osu10670/lsst_stack/python/miniconda3-4.5.4/envs/lsst-scipipe-fcd27eb/lib/python3.6/site-packages/meds/meds.py", line 305, in get_psf
    raise RuntimeError("this MEDS file has no 'psf' extension")
RuntimeError: this MEDS file has no 'psf' extension
